region/tourian/main.json

```diff
@@ -478,8 +478,15 @@
                   "notable": false,
                   "requires": [
                     {"or": [
-                      "ScrewAttack",
-                      "Ice"
+                      "Ice",
+                      {"and": [
+                        "ScrewAttack",
```

Comment:
I wonder if screw should be moved as it's own strat. Someone might be stuck and then see "Easy Evade" and be confused, while having just charge. Also, instead of dodge while shooting, I would think Space+Screw is trivial?